tests/test_core.py

```diff
@@ -7840,7 +7840,7 @@ def setUp(self):
 
     os.chdir(self.get_dir()) # Ensure the directory exists and go there
 
-    self.emcc_args = emcc_args[:]
+    self.emcc_args = emcc_args
```

Comment:
this looks unrelated to the PR title?

(I'm also unsure it is safe, as we may append to that object and so end up having side effects?)